backend/commons/src/test/java/org/sonarsource/sonarlint/core/commons/StringUtilsTests.java

```diff
@@ -20,17 +20,28 @@
 package org.sonarsource.sonarlint.core.commons;
 
 import org.junit.jupiter.api.Test;
-import org.sonarsource.sonarlint.core.commons.util.StringUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.sonarsource.sonarlint.core.commons.util.StringUtils.pluralize;
+import static org.sonarsource.sonarlint.core.commons.util.StringUtils.sanitizeAgainstRTLO;
 
 class StringUtilsTests {
 
   @Test
   void should_pluralize_words() {
-    assertThat(StringUtils.pluralize(0, "word")).isEqualTo("0 word");
-    assertThat(StringUtils.pluralize(1, "word")).isEqualTo("1 word");
-    assertThat(StringUtils.pluralize(2, "word")).isEqualTo("2 words");
+    assertThat(pluralize(0, "word")).isEqualTo("0 word");
```

Comment:
Have you seen the post from JB? Apparently `0 words` should be plural :exploding_head: